UnitTests/GitCommandsTests/Git/GitBlameCommitTests.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Text;
+using GitCommands;
+using NUnit.Framework;
+
+namespace GitCommandsTests.Git
+{
+    [TestFixture]
+    public class GitBlameCommitTests
+    {
+        [Test]
+        public void TestToString()
+        {
+            var committerTime = DateTime.Now;
+            var authorTime = DateTime.Now;
+
+            var str = new StringBuilder();
+
+            str.AppendLine("Author: Author");
+            str.AppendLine("AuthorTime: " + authorTime);
+            str.AppendLine("Committer: committer");
+            str.AppendLine("CommitterTime: " + committerTime);
+            str.AppendLine("Summary: test summary");
+            str.AppendLine();
+            str.Append("FileName: fileName.txt");
+
+            var commit = new GitBlameCommit(
+                "ObjectId",
+                "Author",
+                "author@authormail.com",
+                authorTime,
+                "authorTimeZone",
+                "committer",
+                "committer@authormail.com",
+                committerTime,
+                "committerTimeZone",
+                "test summary",
+                "fileName.txt");
+
+            Assert.AreEqual(str.ToString(), commit.ToString());
```

Comment:
For comparison of sizable chunks of content I've been using [ApprovalTests](https://github.com/approvals/ApprovalTests.Net). May worth considering

Reply:
Will check it out.

Reply:
ApprovalTests looks interesting, thanks. I might be a good fit for metadata-extractor. There's been some discussion about improving regression tests there as there are two implementations (Java/.NET) and one repo of test images to verify against.